deploy/examples/cert-manager/README.md

```diff
@@ -0,0 +1,35 @@
+# Steps
+
+## Install cert-manager
+    helm install \
+        cert-manager jetstack/cert-manager \
+        --namespace cert-manager \
+        --create-namespace \
+        --version v1.6.1 \
+        --set installCRDs=true
+
+## Prepare istio-system Namespace
+
+1. Create system namespace:
+
+        kubectl create ns istio-system
+
+2. Provision certificates:
+
+        kubectl apply -f selfsigned-ca.yaml -n istio-system
```

Comment:
`kubectl apply -f deploy/examples/cert-manager/selfsigned-ca.yaml -n istio-system
`
Please, update this command - it is easy to get the yaml file